### PR TITLE
Update ignore list

### PR DESCRIPTION
**Pull request by AnnaRybkina**
Trying to make the MISSING list a bit shorter

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import drnm from 'drnm'
...
...90', 'backdrop-brightness-95', 'backdrop-brightness-100', 'backdrop-brightness-105', 'backdrop-brightness-110', 'backdrop-brightness-125', 'backdrop-brightness-150', 'backdrop-brightness-200', 'backdrop-contrast-0', 'backdrop-contrast-50', 'backdrop-contrast-75', 'backdrop-contrast-100', 'backdrop-contrast-125', 'backdrop-contrast-150', 'backdrop-contrast-200', 'backdrop-hue-rotate-0', 'backdrop-hue-rotate-15', 'backdrop-hue-rotate-30', 'backdrop-hue-rotate-60', 'backdrop-hue-rotate-90', 'backdrop-hue-rotate-180', '-backdrop-hue-rotate-180', '-backdrop-hue-rotate-90', '-backdrop-hue-rotate-60', '-backdrop-hue-rotate-30', '-backdrop-hue-rotate-15', 'backdrop-invert-0', 'backdrop-invert', 'backdrop-opacity-0', 'backdrop-opacity-5', 'backdrop-opacity-10', 'backdrop-opacity-20', 'backdrop-opacity-25', 'backdrop-opacity-30', 'backdrop-opacity-40', 'backdrop-opacity-50', 'backdrop-opacity-60', 'backdrop-opacity-70', 'backdrop-opacity-75', 'backdrop-opacity-80', 'backdrop-opacity-90', 'backdrop-opacity-95', 'backdrop-opacity-100', 'backdrop-saturate-0', 'backdrop-saturate-50', 'backdrop-saturate-100', 'backdrop-saturate-150', 'backdrop-saturate-200', 'backdrop-sepia-0', 'backdrop-sepia' ]
```
Comment (AnnaRybkina) [inline index.js:9]: Removed `'focus-ring'` from the list. It seems impossible to see the changes in this diff 😅

File: index.js [hunk 2]
```diff
@@ -35,6 +35,10 @@ export const buildList = () => {
       if (s.includes('red')) return
       if (s.includes('bluegray')) return
       if (s.includes('gray')) return
+      if (s.includes('focus-ring')) return
+      if (s.includes('bg-current')) return
+      if (s.includes('bg-white')) return
+      if (s.includes('bg-transparent')) return
```
Comment (AnnaRybkina) [inline index.js:41]: Does these make sense here?

File: index.js [hunk 3]
```diff
@@ -35,6 +35,11 @@ export const buildList = () => {
...
+      if (s.includes('bg-none')) return
```
Comment (AnnaRybkina) [inline index.js:42]: Does these make sense here? I think we will handle the background colors with a help of tokens. But please correct me if I'm wrong

Comment (AnnaRybkina) [reply index.js:42]: `bg-none` sounds though like a different thing

Comment (pearofducks) [reply index.js:42]: Yeah we might have a `bg-none`, but the rest are OK for now - we might need to iterate on these anyhow so no big stress!

File: index.js (hunk 1 above)
Comment (pearofducks) [reply index.js:9]: No prob, why remove `focus-ring`?

Comment (AnnaRybkina) [reply index.js:9]: Ah this comment was very straight forward in my head 🤦‍♀️ I removed it from the list above and added it here instead https://github.com/warp-ds/parity/pull/2/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R38
That is to remove these 👇 from the MISSING list:
```
MISSING focus\:ring:focus
MISSING focus-ring:focus
MISSING focus-ring:focus:not(:focus-visible)
MISSING focus-ring:focus-visible
```

Comment (pearofducks) [reply index.js:9]: Ahh! Makes sense! Good job! 👏

**Review by pearofducks — APPROVED** [review]

**Pull request merged**